protocol/messenger_contacts.go

```diff
@@ -904,30 +899,23 @@ func (m *Messenger) BlockContact(contactID string, fromSyncing bool) (*Messenger
 
 // The same function as the one above.
 // Should be removed with https://github.com/status-im/status-desktop/issues/8805
-func (m *Messenger) BlockContactDesktop(contactID string) (*MessengerResponse, error) {
+func (m *Messenger) BlockContactDesktop(ctx context.Context, contactID string) (*MessengerResponse, error) {
 	response := &MessengerResponse{}
 
-	err := m.blockContact(response, contactID, true, false)
+	err := m.blockContact(ctx, response, contactID, true, false)
 	if err != nil {
 		return nil, err
 	}
 
-	response, err = m.DeclineAllPendingGroupInvitesFromUser(response, contactID)
+	response, err = m.DeclineAllPendingGroupInvitesFromUser(ctx, response, contactID)
 	if err != nil {
 		return nil, err
 	}
 
-	notifications, err := m.persistence.DismissAllActivityCenterNotificationsFromUser(contactID, m.getCurrentTimeInMillis())
+	_, err = m.DismissAllActivityCenterNotificationsFromUser(ctx, contactID, m.GetCurrentTimeInMillis())
```

Comment:
same here, should maybe sync `dismiss`?

Reply:
I see there're more places with `Dismiss...` below. This applies to all cases.

Reply:
It's better to add dismissed notifications to the response here, too.

Reply:
> It's better to add dismissed notifications to the response here, too.

maybe separate PR would be better, this PR mainly focused on #3949 🙂